packages/teams-js/CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
 # Change Log - @microsoft/teams-js
 
-This log was last generated on Wed, 01 Nov 2023 18:15:02 GMT and should not be manually modified.
+This log was last generated on Thu, 30 Nov 2023 23:24:44 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## 2.18.0
+
+Thu, 30 Nov 2023 23:24:44 GMT
+
+### Minor changes
+
+- Extended `RequestAppAudioHandlingParams` by adding `audioDeviceSelectionChangedCallback` for speaker selection updates
+
+### Patches
+
+- Added `meetingStage` and `settings` framecontexts to `clipboard`.
+- Added additional telemetry to `App`, `Dialog`, `GeoLocation`, `Location`, `Navigation`, `Pages`, and `Tasks` capabilities
+- Created new `MessageRequest` interface with required properties to enhance type-safety
+- Added telemetry to `barcode`, `calendar`, `call`, `clipboard`, `mail`, `marketplace` and `media` capabilities
```

Comment:
Not a huge deal but perhaps the line two above this could be combined with this one?

Reply:
Hmm we can make that change, but then the README would be out of sync with the one on the npm package

Reply:
Then don't.  😶